src/igvfd/tests/test_audit_index_file.py

```diff
@@ -18,3 +18,14 @@ def test_audit_index_files(testapp, reference_file, index_file_tbi):
         audit['category'] == 'unexpected indexed file'
         for audit in res.json['audit'].get('ERROR', {})
     )
+    testapp.patch_json(
+        index_file_tbi['@id'],
+        {
+            'derived_from': [tabular_file_bed['@id']]
+        }
+    )
+    res = testapp.get(index_file_tbi['@id'] + '@@audit')
+    assert not any(
```

Comment:
Usually we write this kind of assertion like the case in line 6 above. Is there a reason to use not any() vs all()?

Reply:
fixing this to be
```assert all(
        audit['category'] != 'unexpected indexed file'
        for audit in res.json['audit'].get('ERROR', {})
    )```